### PR TITLE
[RN][New Architecture] Update Playbook with 0.72 changes

### PR DESCRIPTION
**Pull request by cipolleschi**
This PR updates the New Architecture playbook with the changes discussed in 0.72

### CODE REVIEW

File: docs/new-architecture-app-intro.md [hunk 1]
```diff
@@ -19,174 +19,74 @@ This guide is written with the expectation that you’re using the [**latest** R
...
+It’s now time to run your app to verify that everything works correctly:
 
-For example, if you use the template, your project structure for what concerns iOS should look like this:
+```bash
+# To run android
+yarn android
 
+# To run iOS
+yarn ios
 ```
```
Comment (Pranav-yadav) [inline docs/new-architecture-app-intro.md:67]: We should provide `npm run` equivalents wherever possible throughout this diff.
You should use TabItems and Tab components to provide tabs for `npm` and `yarn`

**Review by Pranav-yadav — COMMENTED** [review]
Left a few suggestions around formatting and corrections.

File: docs/new-architecture-app-intro.md [hunk 2]
```diff
@@ -19,174 +19,74 @@ This guide is written with the expectation that you’re using the [**latest** R
...
-```sh
-echo 'export NODE_BINARY=$(command -v node)' > .xcode.env
-```
-
-If you need it, you can also open the file and replace the `$(command -v node)` with the path to the node executable.
-React Native also supports a local version of this file `.xcode.env.local`. This file is not synced with the repository to let you customize your local setup, if it differs from the Continuous Integration or the team one.
+If you successfully updated your project to the latest version of React Native, you **already meet** all the prerequisites to use the New Architecture on iOS.
 
-2. Fix an API change in the `AppDelegate.m`. Open this file and apply this change:
+You will only need to reinstall your pods by runningpod install with the right flag:
```
Comment (cortinico) [inline docs/new-architecture-app-intro.md:50]:
```suggestion
You will only need to reinstall your pods by running `pod install` with the right flag:
```

**Review by cortinico — COMMENTED** [review]
Great cleanup 👍

**Review by cortinico — APPROVED** [review]
LGTM 🚀

File: docs/new-architecture-app-intro.md [hunk 3]
```diff
@@ -19,174 +19,74 @@ This guide is written with the expectation that you’re using the [**latest** R
...
+If you followed the previous steps but your app uses some custom Native Components that have not been migrated to the New Architecture completely, you are going to see some reddish/pinkish boxes saying that the component is not compatible with Fabric. This is happening because custom Native Components written for the legacy architecture can't run as-is in the New Architecture.
 
-```bash
-// Run pod install with the flags
-RCT_NEW_ARCH_ENABLED=1 pod install
-```
+Starting from React Native 0.72, we worked on a interoperability layer to let you use legacy components in the New Architecture without having to wait for them to be migrated.
 
-## Android - Prerequisites
+<!-- TODO: Add link to the WG as soon as we publish it. -->
```
[the remark this answers is not in the document]
Comment (cipolleschi) [reply docs/new-architecture-app-intro.md]: updated

**Pull request merged**